ladybug/src/main/resources/springIbisTestTool.xml

```diff
@@ -141,7 +141,7 @@
 		</property>
 	</bean>
 
-	<bean name="xsltResource" class="java.lang.String" primary="true">
+	<bean name="xsltResource" class="java.lang.String">
```

Comment:
Geen onverwachte fouten bij het creëren van de context als deze niet meer de primary string-bean is?
(Ik weet niet of deze change bewust is meegenomen of per ongeluk is blijven staan na onze probeersels van afgelopen vrijdag).

Reply:
bewust, ik zie niet in waarom deze primary moet zijn? Ik zal het nog even navragen bij @jacodg.

Reply:
Zolang er niks fout gaat bij opstarten van framework en draaien van IAF-Test zal het wel goed zijn.